app/views/content/landing/grow/_content.html.erb

```diff
@@ -56,7 +56,7 @@
       </p>
 
       <ul>
-      <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> within 5 years</li>
+      <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```

Comment:
Figures all look fine, but I think the content looks strange to have two sentences on the bullet point. Normally either all bullet points should be full sentences and be capitalised, or not at all. 
```suggestion
      <li>your salary is reviewed annually and most teachers get a pay rise (depending on your school), so you could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```

Reply:
Added to separate ticket